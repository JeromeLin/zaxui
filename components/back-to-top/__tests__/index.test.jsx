import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import BackToTop from '../index';

describe('Badge', () => {
  it('renders correctly', () => {
    const wrapper = mount(<BackToTop>Up</BackToTop>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
