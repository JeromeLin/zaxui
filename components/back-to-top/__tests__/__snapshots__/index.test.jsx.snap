// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
<BackToTop
  prefixCls="za-back-to-top"
  scrollContainer={[Window]}
  speed={100}
  visibleDistance={400}
>
  <Portal
    containerInfo={
      <div
        class="za-back-to-top-container"
      >
        <div
          class="za-back-to-top"
          style="display: none;"
        >
          Up
        </div>
      </div>
    }
  >
    <div
      className="za-back-to-top"
      onClick={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
    >
      Up
    </div>
  </Portal>
</BackToTop>
`;
